.github/workflows/build-images-base.yaml

```
@@ -63,7 +67,7 @@ on:
         type: string
 
 env:
-  IMG_TAGS: ${{ github.sha }} ${{ inputs.kuadrantOperatorTag }}
+  IMG_TAGS: ${{ inputs.kuadrantOperatorTag }} ${{ inputs.additionalTags == null && '' || inputs.additionalTags }}
```

I'd leave it as...

```suggestion
  IMG_TAGS: ${{ inputs.kuadrantOperatorTag }} ${{ inputs.additionalTags }}
```

since the default is an empty string, no damage done for the rest of the consumers of this workflow base